usr/sbin/genRegionServerCert

```diff
@@ -178,18 +180,22 @@ create_self_signed_cert(
     options
 )
 
-os.system(
-    'cp /root/regionServCert/%s /root/regionServCert/%s.pem'
-    % (cert_file, options.hostname)
-)
-os.rename(
-    '/root/regionServCert/%s' % cert_file,
-    '/etc/apache2/ssl.crt/%s' % cert_file
-)
-os.rename(
-    '/root/regionServCert/%s' % key_file,
-    '/etc/apache2/ssl.key/%s' % key_file
-)
+shutil.copy(
+        '/root/regionServCert/%s' % cert_file,
+        '/root/regionServCert/%s.pem' % options.hostname
+    )
+
+for filename in [cert_file, key_file]:
+    source_file = '/root/regionServCert/%s' % filename
+    destination = '/etc/apache2/ssl.crt/%s' % filename
+    try:
+        os.rename(source_file, destination)
+    except OSError as err:
+        # OSError: [Errno 18] Invalid cross-device link
+        if err.errno == errno.EXDEV:
+            # best effort across file systems
+            shutil.copyfile(source_file, destination)
```

Comment:
Note: copyfile doesn't copy permissions if that matters.

Reply:
Note: if we are running in Python3 shutil.move should work too: https://docs.python.org/3/library/shutil.html#shutil.move